tests/unit/null_communities.c

```diff
@@ -85,7 +85,7 @@ int main(void) {
 
     igraph_vector_int_resize(&membership, 1);
 
-    igraph_community_label_propagation(&g, &membership, IGRAPH_ALL, NULL, NULL, NULL);
+    igraph_community_label_propagation(&g, &membership, IGRAPH_ALL, NULL, NULL, NULL, IGRAPH_LPA_DOMINANCE);
```

Comment:
Are you sure we don't need extra lines for all other variants here? It would be simple to add them. From my experience, proper null graph handling tends to break easily when refactoring code.

Reply:
Sure.